Makefile

```diff
@@ -40,35 +40,43 @@ help:
 	@echo "$$USAGE"
 
 repo:
-	./scripts/repo-build package build/package build/repo "$$remote_repo"
+	./scripts/repo_build.py $(FLAGS)
 
 repo-local:
-	./scripts/repo-build -l package build/package build/repo "$$remote_repo"
+	./scripts/repo_build.py --local
 
 repo-new:
-	./scripts/repo-build -n package build/package build/repo "$$remote_repo"
+	./scripts/repo_build.py --no-fetch $(FLAGS)
 
 repo-check:
 	./scripts/repo-check build/repo
 
 $(RECIPES): %:
-	./scripts/package-build package/"${@}" build/package/"${@}"
+	./scripts/package_build.py $(FLAGS) "$(@)"
 
-$(RECIPES_PUSH): %:
-	ssh root@"${HOST}" mkdir -p .cache/opkg
-	scp build/package/"$(@:%-push=%)"/*/*.ipk root@"${HOST}":.cache/opkg
+push: %:
```

Comment:
how does this fit with local development? is your rM setup to point at .cache/toltec as a repository or do you manually install packages out of .cache/toltec?

Reply:
This pushes the whole repo, including the Packages index, so that you can add /home/root/.cache/toltec as a local repository in `opkg.conf`. The downside is that since it’s using scp instead of rsync, it pushes the whole set of packages everytime. Couldn’t get rsync to work in the PC→rM direction.

Reply:
`rsync --rsync-path /opt/bin/rsync blah blah`

Reply:
Not sure how I missed that flag. Thanks for the heads-up!